SessionHandlerRedis.module

```diff
@@ -153,7 +159,8 @@ class SessionHandlerRedis extends WireSessionHandler implements Module, Configur
     public function read($id)
     {
         $key = $this->key($id);
-        $data = $this->redis->get($key);
+        for ($i = 0; $i < 2 && !($data = $this->redis->get($key)); $i++)
+            $this->write($id, TRUE);
```

Comment:
That's interesting. Is there a read-before-write going on at a higher level. Still, should be OK.